examples/src/test/scala/docs/eventsourced/ShoppingCart.scala

```diff
@@ -135,7 +135,7 @@ object ShoppingCart {
   final case class CheckedOut(cartId: String, eventTime: Instant) extends Event
 
   //#slicingTags
-  val tags = Vector("carts-0", "carts-1", "carts-2", "carts-3", "carts-4")
+  val tags = Vector.tabulate(5)(i => s"carts-$i")
```

Comment:
_fancy_

Reply:
synced with what we have in the Microservices tutorial